tests/ftqc/test_ftqc_lattice.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=no-name-in-module, no-self-use, protected-access
+"""Unit tests for the lattice module"""
+
+import networkx as nx
+import pytest
+
+from pennylane.ftqc import Lattice, generate_lattice
+
+
+class TestLattice:
+    """Test for the qml.ftqc.Lattice class."""
+
+    def test_lattice_creation_with_graph(self):
+        """Test for Lattice object created by a nx.Graph object."""
+        graph = nx.Graph([(0, 1), (1, 2)])
+        lattice = Lattice("test", graph=graph)
+        assert lattice.shape == "test"
+        assert len(lattice.get_nodes()) == 3
+        assert len(lattice.get_edges()) == 2
```

Comment:
There are several places where we still call these even though they are properties - technically it's not broken, but I think it's cleaner to avoid it.